index.html

```diff
@@ -1147,7 +1207,7 @@ <h3>Framing Algorithm</h3>
 
 <p>The recursive algorithm operates with a <a>framing state</a> (<var>state</var>),
   created initially using
-  the <a>object embed flag</a> set to <code>true</code>,
+  the <a>object embed flag</a> set to <code>@always</code>,
```

Comment:
Note that this is a change from the current algorithm. `true` mapped to `@last`. Is this intentional? If so, it will make the default behavior of framing potentially add data to the output -- which is something we avoided in the past.

Reply:
Perhaps it should be `@last`? The Boolean values were holdovers and I don’t think they make as much sense going forward.

Reply:
Yes, it should be `@last` to keep the previous behavior. To be clear, I'm fine with removing the boolean values -- I just wanted to make sure we weren't making an unintentional change here. If we change this to `@last` it should keep the same behavior as before. If `@first` becomes a thing and provides significant performance improvements, we should consider switching this to `@first`.